src/common-opts.ts

```diff
@@ -0,0 +1,39 @@
+import os from "os";
```

Comment:
Same here, I don't think there are default exports for `os` and `path`

Reply:
`allowSyntheticDefaultImports` is enabled which allows for a default export, even if there isn't one. See https://www.typescriptlang.org/docs/handbook/compiler-options.html

Reply:
Ah, I see, thanks